### PR TITLE
chore: add simple readme for generator

### PR DESCRIPTION
**Pull request by codyoss**
Fixes: #2626

### CODE REVIEW

**Review by tritone — APPROVED** [review]
Thanks for adding this, very helpful!

File: google-api-go-generator/README.md
Comment (Shanzay009) [inline google-api-go-generator/README.md:1]: [Uploading WhatsApp Chat - Saqib .zip…]()

Comment (Shanzay009) [reply google-api-go-generator/README.md:1]: > Uploading WhatsApp Chat - Saqib .zip…

❌

**Review by quartzmo — APPROVED** [review]
Thank you!

**Pull request merged**